backend/app/store/image/bolt_store.go

```diff
@@ -60,16 +60,14 @@ func NewBoltStorage(fileName string, maxSize int, maxHeight int, maxWidth int, o
 	}, nil
 }
 
-// Save data from reader to staging bucket in DB
-func (b *Bolt) Save(fileName string, userID string, r io.Reader) (id string, err error) {
+// SaveWithID saves data from reader with given id
+func (b *Bolt) SaveWithID(id string, r io.Reader) (string, error) {
```

Comment:
I haven't checked, but aren't these changes should have their counterpart in `_example/memory_store`?

Similar changes for me required them.

Reply:
As far as I recall I have never introduced remote to image.store, and as of now it has no way to be customized. Adding remote the same way we do for both store.admin and store.engine is a good thing to have, but should be handled in a separate tiicket/pr.

Reply:
@umputun could you please create a PR about that? I don't have enough context to do that.

Reply:
#572 created, resolving this thread.